backend/app/adapter/db/url.go

```diff
@@ -96,6 +96,21 @@ WHERE "%s"=$1;`,
 	return url, nil
 }
 
+// GetByAliases finds URLs for a list of aliases
+func (u URLSql) GetByAliases(aliases []string) ([]entity.URL, error) {
+	urls := make([]entity.URL, 0)
```

Comment:
I think it is a good practice to respect [CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices)

Reply:
will change to that